inc/parser/code.php

```diff
@@ -25,10 +25,16 @@ public function code($text, $language = null, $filename = '') {
         $filename = \dokuwiki\Utf8\Clean::stripspecials($filename, '_');
 
         // send CRLF to Windows clients
-        if(strpos($INPUT->server->str('HTTP_USER_AGENT'), 'Windows') !== false) {
-            $text = str_replace("\n", "\r\n", $text);
+        if(strpos($INPUT->server->str('HTTP_USER_AGENT'), 'Windows') !== false && $linefeed !== "Unix") {
```

Comment:
I guess $linefeed is not defined.

Reply:
resolved, the function now has the linefeed as an argument and it defaults to ''